test/integration/model/create.test.js

```diff
@@ -1351,6 +1351,29 @@ describe(Support.getTestDialectTeaser('Model'), function() {
       });
     });
 
+    it('should not insert NULL for unused fields', function () {
+      var Beer = this.sequelize.define('Beer', {
+          style: Sequelize.STRING,
+          size: Sequelize.INTEGER,
+      });
+
+      return Beer.sync({force: true}).then(function () {
+        return Beer.bulkCreate([{
+            style: 'ipa'
+        }], {
+          logging: function(sql) {
+            if (dialect === 'postgres') {
```

Comment:
Hm while thinking about it: A unit test would not help here. It's not really about the SQL generation but what happens earlier on. I could refactor the bulkInsertQuery, but I doubt that this would be the wanted solution.